objects/UniversalBenefit.js

```diff
@@ -144,9 +146,9 @@ export function CreateBenefitObjWithCPPData(cppBenefitData) {
     [
       CreateBenefitSummary(
         SummaryTypes.PaymentAmount,
-        cppBenefitData.netAmount
+        '$' + cppBenefitData.netAmount
```

Comment:
The $ sign would need to go after the number in french. 
We probably want to handle this on the component. You can see how it's done here as an example https://github.com/DTS-STN/sc-digital-centre/blob/cc0046397c76bb8b24b588feba157a7fc4e3ccec/components/atoms/BenefitCardHeaderActive.js#L90